packages/@lightningjs/ui-components/src/components/CardContent/CardContent.d.ts

```diff
@@ -17,31 +17,87 @@
  */
 
 import lng from '@lightningjs/core';
-import { StylePartial } from '../../types/lui';
+import { Color, StylePartial } from '../../types/lui';
 import Card, { CardStyle } from '../Card';
-import { default as MetadataCardContent } from '../MetadataCardContent';
+import TextBox, { TextBoxStyle } from '../TextBox';
+import MetadataBase from '../MetadataBase';
 
 type ImageSize = {
   w: number;
   h: number;
 };
 
-export type CardContentStyle = CardStyle & {
+type CardContentStyle = CardStyle & {
+  backgroundColor: Color;
   expandedW: number;
   expandedH: number;
   imageSize: ImageSize;
+  metadata: TextBoxStyle;
 };
 
-export default class CardContent extends Card {
+declare namespace CardContent {
+  export interface TemplateSpec extends Card.TemplateSpec {
+    /**
+     * when collapseToMetadata is true and shouldCollapse is enabled, the image collapses to show the metadata
+     */
+    collapseToMetadata?: boolean;
+    /**
+     * object containing all properties supported in the MetadataCardContent component
+     */
+    metadata?: object;
```

Comment:
this should be updated to a MetadataCardContent patch object once it's d.ts has been updated